webpack/assets/javascripts/react_app/components/LoginPage/LoginPage.scss

```diff
@@ -0,0 +1,19 @@
+.login-pf {
```

Comment:
why this CSS is not part of patternfly?

Reply:
small styling that shouldn't affect all patternfly consumers,
such as background and caption.